src/coreclr/gc/gc.h

```diff
@@ -348,8 +348,8 @@ inline bool IsServerHeap()
 #define MAX_LONGPATH 1024
 #endif // MAX_LONGPATH
 
-// #define TRACE_GC
-// #define SIMPLE_DPRINTF
+ //#define TRACE_GC
+ //#define SIMPLE_DPRINTF
```

Comment:
If you need to make another change anyway, you could put these spaces back to reduce the churn

Reply:
oh that's totally not intentionally. I use VS code for CR locally and apparently it ignores white space changes by default.. so it simply didn't show this change to me. will revert it.